core/src/main/scala/org/typelevel/otel4s/otel4s.scala

```diff
@@ -30,7 +31,17 @@ sealed trait AttributeKey[A] {
 }
 object AttributeKey {
   private class Impl[A](val name: String, val `type`: AttributeType[A])
-      extends AttributeKey[A]
+      extends AttributeKey[A] {
+    override final def toString: String = Show[AttributeKey[A]].show(this)
```

Comment:
Unfortunately, it's a needed one. Otherwise the `Impl#toString` is rendered as `org.typelevel.otel4s.AttributeKey$Impl@4183c10f`.

Reply:
I usually put the implementation in the Java methods and have the Cats instances delegate to those, but it's not a big deal either way.  I definitely don't want to block this any longer.